redaxo/src/core/lib/sql/table.php

```diff
@@ -135,18 +135,24 @@ private function __construct($name, int $db = 1)
             WHERE c.constraint_schema = DATABASE() AND c.table_name = ?', [$name]);
         $foreignKeys = [];
         foreach ($foreignKeyParts as $part) {
-            $foreignKeys[$part['constraint_name']][] = $part;
+            // since mysql8  upper-case keys are returned
+            $foreignKeys[$part['constraint_name'] ?? $part['CONSTRAINT_NAME']][] = $part;
```

Comment:
Würde es auch gehen, schon in der Query `c.constraint_name as constraint_name` zu notieren?
Oder würde man mit `SELECT c.CONSTRAINT_NAME` ggf. in allen DBs die großgeschriebene Variante bekommen?
Denn bei mir scheinen die Spalten eigentlich auch großgeschrieben zu sein, und scheinbar kommen sie nur klein an, da es in der Query zu notiert ist. (Und das Verhalten ist scheinbar in 8 nicht mehr so)

Reply:
gerne testen.. ich hab lokal kein mysql8 und teste nur in travis

Reply:
Klappt.